test/xpu_api/containers/sequences/array/array.data/data_const.pass.cpp

```diff
@@ -62,6 +62,7 @@ main()
                     typedef dpl::array<T, 0> C;
                     const C c = {};
                     const T* p = c.data();
+                    (void)p;
```

Comment:
May we use here `[[maybe_unused]]` instead of this new line?

Reply:
There is another case of (void)p; above, so I used that for consistency.

Reply:
OK